### PR TITLE
Update whats-new-in-preview.md

### PR DESCRIPTION
**Pull request by alhopper-msft**
Typo fixes

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/whats-new-in-preview.md [hunk 1]
```diff
@@ -28,8 +28,8 @@ Want to know how to participate in the preview? Checkout [About the Microsoft 36
...
 - **Billing account** - Targeted release customers will see the new Billing account page first and then we'll roll it out to world-wide. When it's available for you, go to **Billing** > **Billing account**. What can you do on the new billing account page? I'm glad you asked:
-    - Update the address and other contat information in your organizational profile directly from this page. You don't have to go to **Settings** > **Organization profile**, unless you want to.
-    - And we're making life easier for Direct or Volume licensing customers, you can accept and review customer agreements from **Billing accounts**. You can also connect with other orgs (that you manage?) allowing you to link the orgs together to share licenses and resources.
+    - Update the address and other contact information in your organizational profile directly from this page. You don't have to go to **Settings** > **Organization profile**, unless you want to.
```
Comment (JohanFreelancer9) [inline Office365-Admin/whats-new-in-preview.md]:
```suggestion
    - Update the address and other contact information in your organizational profile directly from this page. There is no need to go to **Settings** > **Organization profile**, unless you want to.
```

File: Office365-Admin/whats-new-in-preview.md [hunk 2]
```diff
@@ -28,8 +28,8 @@ Want to know how to participate in the preview? Checkout [About the Microsoft 36
...
+    - And we're making life easier for Direct or Volume licensing customers, you can accept and review customer agreements from **Billing accounts**. You can also link together orgs that you manage to share licenses and resources.
```
Comment (JohanFreelancer9) [inline Office365-Admin/whats-new-in-preview.md]:
```suggestion
    - And we're also making life easier for Direct or Volume licensing customers. You can now accept and review customer agreements from **Billing accounts**. You can also link together orgs that you manage to share licenses and resources.
```

Comment (JohanFreelancer9) [inline Office365-Admin/whats-new-in-preview.md]:
```suggestion
    - And we're making life easier for Direct or Volume licensing customers. You can now accept and review customer agreements from **Billing accounts**, and link together orgs that you manage to share licenses and resources.
```

**Comment by mypil** [comment]
@alhopper-msft - we need your kind assistance to merge the new commit for the copy/edit review made for this PR you created. Can you help with this please? Thank you.

**Comment by mypil** [comment]
@adefowler - Can you please assist to check, approve and merge the content of this PR?

Thank you.

**Comment by adefowler** [comment]
What document? There isn’t enough information to act on.

**Comment by mypil** [comment]
@adefowler - I believe @alhopper-msft is referencing the typo errors on this article link - https://docs.microsoft.com/en-us/office365/admin/whats-new-in-preview?view=o365-worldwide

**Comment by adefowler** [comment]
Sorry, I had to close the pull request. There have been too many changes to the article since these changes were made. I’ll fix the typos manually.

**Comment by mypil** [comment]
@adefowler - This is noted. Thank you!

**Pull request closed without merging**